### PR TITLE
Catalan Translation (#439)

### PR DESCRIPTION
**Pull request by olegkorol**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | none |
| Bug fix?         | no      |
| New feature?     | yes      |
| Includes tests?  | no      |
| All Tests pass?  | -      |
| Related ticket?  | #439        |

---

## Description

Added a Catalan translation :)

I wasn't able to run `$ yarn contributors:add` though, as it exits with errorcode 1 :/

### CODE REVIEW

**Comment by chinesedfan** [comment]
> I wasn't able to run $ yarn contributors:add though, as it exits with errorcode 1 :/

Can you paste your error logs? Maybe we can help you.

**Comment by olegkorol** [comment]
Thanks @chinesedfan, here are the logs:
```
yarn contributors:add

yarn run v1.10.1
$ all-contributors add
? What is the contributor's null username? olegkorol
? What are the contribution types? 🌍  Translation
Cannot read property 'then' of null
error Command failed with exit code 1.
```

**Comment by olegkorol** [comment]
I've added an entry to the contributors list manually, @housseindjirdeh :)

File: .all-contributorsrc [hunk 1]
```diff
@@ -842,6 +842,15 @@
       "contributions": [
         "code"
       ]
+    },
```
Comment (chinesedfan) [inline .all-contributorsrc]: @olegkorol `.all-contributorsrc` is broken in previous PRs, which is not a valid JSON. That's why all-contributors cli is failed. Please add a left brace to match this right brace. Thanks!

Comment (olegkorol) [reply .all-contributorsrc]: Done ;)

Comment (olegkorol) [reply .all-contributorsrc]: @chinesedfan

**Comment by olegkorol** [comment]
@lex111 
Sorry, my bad. I forgot to add an entry on that file. Should be ok now.

File: src/auth/screens/language-settings.js [hunk 2]
```diff
@@ -99,6 +99,11 @@ export default [
     emojiCode: ':flag-tw:',
     name: '正體中文',
   },
+  {
+    code: 'ca',
```
Comment (lex111) [inline src/auth/screens/language-settings.js]: Alphabetically must be..

Comment (olegkorol) [reply src/auth/screens/language-settings.js]: Oh, I saw the sorting function after the array, so I though the order doesn't matter :)
But I'm going to edit this now, thanks for the feedback.

Comment (olegkorol) [reply src/auth/screens/language-settings.js]: @lex111

Comment (lex111) [reply src/auth/screens/language-settings.js]: Ooops :)

then it doesn't matter, I just thought that the code should be consistent...

Comment (lex111) [reply src/auth/screens/language-settings.js]: Hmm, I just got all the latest changes from the master branch and tried to build the application, everything was OK
MacOS High Sierra

**Comment by lex111** [comment]
@olegkorol great work, thank you! Happy Hacktoberfest! 🎃

**Pull request merged**